forms.py

```diff
@@ -0,0 +1,10 @@
+from flask_wtf import Form
+from wtforms.fields import StringField, SubmitField, BooleanField, RadioField
+from wtforms.validators import Required
+
+
+class LoginForm(Form):
+    """Accepts a nickname and a room."""
+    name = StringField('Name', validators=[Required()])
+    room = StringField('Room', validators=[Required()])
+    submit = SubmitField('Enter Chatroom')

```

Comment:
This looks like its the forms.py from app/main with no real changes, I'm removing this from the pull requests, if it is needed resubmit with the correct path.  thanks